BLL/Services/DrinkSearchService.cs

```diff
@@ -98,39 +97,25 @@ public IEnumerable<Drink> SearchByIngredients(SortedSet<string> ingredientsToSea
             return drinksFound;
         }
 
-        public IQueryable<Drink> SearchByAlcoholContent(bool alcoholics, bool nonAlcoholics, bool optionalAlcoholics, List<Drink> drinks)
+        public IEnumerable<Drink> SearchByAlcoholContent(bool alcoholics, bool nonAlcoholics, bool optionalAlcoholics)
         {
+            var drinks = _repository.GetAllDrinksAsQueryable();
             var contemporaryList = new List<Drink>();
             if (alcoholics)
             {
-                contemporaryList.AddRange(drinks.Where(x => x.AlcoholicInfo == DrinkIsAlcoholic).ToList());
+                contemporaryList.AddRange(drinks.Where(x => x.AlcoholicInfo == DrinkIsAlcoholic));
             }
 
             if (nonAlcoholics)
             {
-                contemporaryList.AddRange(drinks.Where(x => x.AlcoholicInfo == DrinkIsNonAlcoholic).ToList());
+                contemporaryList.AddRange(drinks.Where(x => x.AlcoholicInfo == DrinkIsNonAlcoholic));
             }
 
             if (optionalAlcoholics)
             {
-                contemporaryList.AddRange(drinks.Where(x => x.AlcoholicInfo == DrinkIsOptionalAlcohol).ToList());
+                contemporaryList.AddRange(drinks.Where(x => x.AlcoholicInfo == DrinkIsOptionalAlcohol));
             }
-            return contemporaryList.AsQueryable();
-        }
-
-
-        public IEnumerable<Drink> SortDrinks(string sortOrder, IEnumerable<Drink> drinks)
-        {
-            switch (sortOrder)
-            {
-                case "name_desc":
-                    drinks = drinks.OrderByDescending(s => s.Name);
-                    break;
-                default:
-                    drinks = drinks.OrderBy(s => s.Name);
-                    break;
-            }
-            return drinks;
```

Comment:
I've deleted this piece of code because it wasn't working as it should - instead I added OrderBy in GetAllDrinks()